webhook/admission/README.md

```diff
@@ -0,0 +1,8 @@
+# Admission Webhook
+
+The admission webhook validates or changes the values in a request.
+An resource request from longhorn-manager might be handled by an
+admission webhook on another node. Therefore, any manipulation of
+the resources of a node, such as creating and deleting folders,
+should be avoided. The manipulations should be handled by the 
+longhorn-manager daemon or controller.
```

Comment:
How about

`
The admission webhook only validates or mutates the spec of hooked resources before resource reconsilation. Beyound that, any resource reconsilation to ensure resources behave as expected should still be handled in the backend (ex: controllers or even host operations like creating/deleting folders at host) as usual. 
`

Reply:
Looks good, thank you.

Reply:
LGTM